### ISSUE TITLE
Freeze at Frame #: 1 

### ISSUE THREAD

**Issue by nbourre** [theAIGuysCode/yolov4-deepsort#119]
I've tried to run the command

`python object_tracker.py --weights ./checkpoints/yolov4-tiny-416 --model yolov4 --video ./data/video/test.mp4 --output ./outputs/tiny.avi --tiny`

It takes an eternity to get to the print "Frame #:1" and it seems to freeze at the first frame. Nothing is showing.

Can someone guide me in the right direction?

I have a RTX 3600.

**Comment by PrakharPrashantSingh** [comment]
Yes there is something wrong with RTX 3000 series.. i have RTX 3070, do one thing, uninstall Tensorflow GPU and Install the new version, it will work!!
I.e.- Pip uninstall tensorflow-gpu

Pip install tensorflow-gpu

Done

**Comment by alvarojsj** [comment]
Any updates on that? I got the same issue and it's really annoying...

**Comment by powerman987** [comment]
> Any updates on that? I got the same issue and it's really annoying...

I wait it for 30min .  and it worked. but the result is not ideal

**Comment by alvarojsj** [comment]
When I wait for some time the video gets processed, but with no detections...

**Comment by powerman987** [comment]
I think the calculation is too large, resulting freeze at Frame #1
1. First ,I tryed tensorflow-cpu ,if it worked. I can confirm the .pb file is fine.
2. And I change my tensorflow-gpu version from 2.3.0 to 2.4.0 .It  worked.
